BigriverBookstore-appleTests/BigriverBookstore_appleTests.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  BigriverBookstore_appleTests.swift
+//  BigriverBookstore-appleTests
+//
+//  Created by Viorel Porumbescu on 04/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+
+import XCTest
+@testable import BigriverBookstore_apple
+
+class BigriverBookstore_appleTests: XCTestCase {
+
```

Comment:
@c-Viorel No tests yes? :) At this point, is it possible to mock the API and to test the retrieveBookList?